src/Command/Service/CreateCoreQueryArchCommandService.php

```diff
@@ -0,0 +1,439 @@
+<?php
+
+namespace Centreon\Command\Service;
+
+use Symfony\Component\Console\Output\OutputInterface;
+use Centreon\Command\Model\ModelTemplate\ModelTemplate;
+use Centreon\Command\Model\DtoTemplate\ResponseDtoTemplate;
+use Centreon\Command\Model\FactoryTemplate\FactoryTemplate;
+use Centreon\Command\Model\PresenterTemplate\PresenterTemplate;
+use Centreon\Command\Model\UseCaseTemplate\QueryUseCaseTemplate;
+use Centreon\Command\Model\RepositoryTemplate\RepositoryTemplate;
+use Centreon\Command\Model\ControllerTemplate\QueryControllerTemplate;
+use Centreon\Command\Model\PresenterTemplate\PresenterInterfaceTemplate;
+use Centreon\Command\Model\RepositoryTemplate\RepositoryInterfaceTemplate;
+
+class CreateCoreQueryArchCommandService extends CreateCoreArchCommandService
```

Comment:
I would rather use composition over inheritance here for the services.

Reply:
Done